lxqt-admin-user/lxqt-admin-user-helper

```diff
@@ -2,10 +2,21 @@
 case "$1" in
 # we only allow executing these commands
 useradd|usermod|userdel|groupadd|groupmod|groupdel|passwd|gpasswd)
-    # TODO: platforms using different commands can add wrapper scripts here.
     export LC_ALL=C
     exec "$@"
     ;;
+pw)
```

Comment:
Thats a good idea, I've used the rename argument to install PROGRAM in cmake, so it chooeses either from .default or .freebsd